db/migrations/20190301141224_create_ilk_state_history_table_and_triggers.sql

```diff
@@ -1,5 +1,5 @@
 -- +goose Up
-CREATE TABLE api.historical_ilk_state
+CREATE TABLE api.ilk_snapshot
```

Comment:
I dig the renaming, and am wondering whether we might want to go ahead and apply it to all of the other `historical_state` tables for consistency